certs/Readme.md

```diff
@@ -0,0 +1,2 @@
+# INSECURE TEST KEYS!
+These keys are for test purposes only. They have been **published in the repo** and must not be relied on for any actual security.
```

Comment:
```suggestion
These keys are for test purposes only. They have been **published in the repo** and must not be relied on for any actual security.

The test key starts with `MIIJQwIBADANBgkqhkiG9w0BAQEFAASCCS0wggkpAgEAAoICAQDNjj6U4Fq3wy81` and the test certificate starts with `MIIFkTCCA3mgAwIBAgIJAJlSoRpOabaGMA0GCSqGSIb3DQEBCwUAMF8xCzAJBgNV`.
``` We should specify what the test keys actually are, in case users accidentally confuse their own for ours.